inc/l10n/namespace.php

```diff
@@ -326,7 +326,7 @@ function wplang_codes() {
  */
 function get_locale() {
 	// If the user has set a locale, use it
-	if ( function_exists( 'wp_get_current_user' ) ) {
+	if ( function_exists( 'wp_get_current_user' ) && is_admin() ) {
```

Comment:
@greatislander You willing to help review this PR? @richard015ar discovered that this is probably the change we need, rather than adding the strings to pressbook-book, etc. My understanding is that this will only load the user's locale on WordPress admin pages -- otherwise it will use the global local (set at the level of the site). Helps us make cleaner distinction between front end and back end localization and avoids strings being translated unneccessarily in the front end when a user who has set their language to something other than a book's language views home page while logged in.